src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -816,8 +858,8 @@ InferenceEngine::Blob::Ptr InferRequest::GetBlob(const std::string& name) {
                 }
 
                 _outputs[name] = data;
-                if (!isDynamic && !externalPtr.count(name) &&
-                    data->getTensorDesc() == MemoryDescUtils::convertToTensorDesc(output->second->getParentEdgesAtPort(0)[0]->getMemory().getDesc())) {
+                if (!externalPtr.count(name) &&
+                    (isDynamic ||(data->getTensorDesc() == MemoryDescUtils::convertToTensorDesc(output->second->getParentEdgesAtPort(0)[0]->getMemory().getDesc()) && !isDynamic))) { // TODO: handle desc incompatible if isDynamic.
```

Comment:
```suggestion
                    (isDynamic || (data->getTensorDesc() == MemoryDescUtils::convertToTensorDesc(output->second->getParentEdgesAtPort(0)[0]->getMemory().getDesc())) { // TODO: handle desc incompatible if isDynamic.
```
We do not need additional `!isDynamic` check since `||` operator is used.

Reply:
It would throw an exception in the desc compatibility check if it is a dynamic case.